jogo42-cb/main.c

```diff
@@ -18,6 +18,16 @@ void MoverJog(GameState* gs);
 // Move os obstaculos
 void MoverObst(GameState* gs);
 
+const float dis=100.0;
+
+bool ataque (GameState* gs)
```

Comment:
Pela convenção de estilo atualmente sendo usada no projeto, funções devem ter nomes com iniciais maiúsculas em cada palavra.

```suggestion
bool Ataque (GameState* gs)
```